nebu/cli/assemble.py

```diff
@@ -54,7 +54,7 @@ def assemble(ctx, input_dir, output_dir):
 
     if collection_assembled_xhtml.exists():
         confirm_msg = (
-            "This will remove '{}', continue?"
+            "Directory '{}' already exists. Would you like to replace it?"
             .format(collection_assembled_xhtml)
```

Comment:
It's not actually a directory that is being replaced here. It used to be the case that we were removing the entire directory. Karen had in later adjustments changed it to only replacing the assembled html file.

Reply:
Oooh... does it mean that we don't need to fix the issue anymore?

Reply:
That could be the case... it's currently saying that it's removing `collection.assembled.xhtml`, which one could imply things about because of the action being taken, but the "Would you like to replace it?" part is good. 👍